<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sm3_veri</class>
 <widget class="QMainWindow" name="sm3_veri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>验证签名</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="CustomLineEdit" name="lineEdit_dir_raw_file">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sm3veri_cho_raw_file">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择原文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_signer_id">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>123hello#@</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>签名者标识：</string>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_dir_sig_file">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sm3veri_cho_sig_file">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择签名文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_dir_pub_file">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sm3veri_cho_pub_file">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择公钥文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sm3veri_verification">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>验签</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_另存为.png</normaloff>:/icons/icons/-_另存为.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
